kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -89,10 +89,23 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
     if (session == null) {
       throw KyuubiSQLException(s"Invalid $sessionHandle")
     }
+    deleteOperationLogSessionDir(sessionHandle)
```

Comment:
When the `closeSession` method is called, the operation is not necessarily close. At this time, there are log files, and the log may also increase.

When `session.close()`, it will close the operation first.

So it is suggested here that `deleteOperationLogSessionDir` should be placed after `session.close()`
Also because `Files.deleteIfExists` can only delete empty folders, consider using `org.apache.kyuubi.Utils#deleteDirectoryRecursively`.

Reply:
my bad, previous I only considered the empty session case which a session is created without operators